<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>555</width>
    <height>444</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3" stretch="0">
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="maximumSize">
      <size>
       <width>10000</width>
       <height>10000</height>
      </size>
     </property>
     <property name="currentIndex">
      <number>1</number>
     </property>
     <widget class="QWidget" name="tab">
      <attribute name="title">
       <string> 串口助手 </string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="0" rowspan="6">
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout">
           <item>
            <widget class="QLabel" name="label">
             <property name="text">
              <string>端口号：</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QComboBox" name="comboBox"/>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_2">
           <item>
            <widget class="QLabel" name="label_2">
             <property name="text">
              <string>波特率：</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QComboBox" name="baud">
             <item>
              <property name="text">
               <string>9600</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>300</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>1200</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>2400</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>4800</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>14400</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>19200</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>24800</string>
              </property>
             </item>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_3">
           <item>
            <widget class="QLabel" name="label_3">
             <property name="text">
              <string>数据位：</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QComboBox" name="bit">
             <item>
              <property name="text">
               <string>8</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>5</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>6</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>7</string>
              </property>
             </item>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_4">
           <item>
            <widget class="QLabel" name="label_4">
             <property name="text">
              <string>校验位：</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QComboBox" name="cbit">
             <item>
              <property name="text">
               <string>0</string>
              </property>
             </item>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_5">
           <item>
            <widget class="QLabel" name="label_5">
             <property name="text">
              <string>停止位：</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QComboBox" name="stopbit">
             <item>
              <property name="text">
               <string>1</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>0</string>
              </property>
             </item>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <spacer name="verticalSpacer">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="close_port">
           <property name="text">
            <string>关闭串口</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="open_port">
           <property name="text">
            <string>打开串口</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="3" column="1">
        <widget class="QLabel" name="label_7">
         <property name="text">
          <string>发送窗口</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1" colspan="2">
        <widget class="QTextEdit" name="receive_window"/>
       </item>
       <item row="0" column="1">
        <widget class="QLabel" name="label_6">
         <property name="text">
          <string>接收窗口</string>
         </property>
        </widget>
       </item>
       <item row="4" column="1" colspan="2">
        <widget class="QTextEdit" name="send_window"/>
       </item>
       <item row="5" column="1" colspan="2">
        <layout class="QHBoxLayout" name="horizontalLayout_7">
         <item>
          <widget class="QCheckBox" name="checkBox">
           <property name="text">
            <string>自动发送</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="send_button">
           <property name="text">
            <string>发送数据</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="clear_button2">
           <property name="text">
            <string>清空数据</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="txt_mod2">
           <property name="text">
            <string>文本模式</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="2" column="1" colspan="2">
        <layout class="QHBoxLayout" name="horizontalLayout_6">
         <item>
          <widget class="QPushButton" name="updat">
           <property name="text">
            <string>更新数据</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="clear_button1">
           <property name="text">
            <string>清空数据</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="txt_mod1">
           <property name="text">
            <string>文本模式</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab2">
      <attribute name="title">
       <string> 数据展示</string>
      </attribute>
      <widget class="QWidget" name="show_p" native="true">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>40</y>
         <width>331</width>
         <height>311</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label_8">
       <property name="geometry">
        <rect>
         <x>120</x>
         <y>10</y>
         <width>121</width>
         <height>20</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>人数分布示意图</string>
       </property>
      </widget>
      <widget class="QLCDNumber" name="tem">
       <property name="geometry">
        <rect>
         <x>400</x>
         <y>80</y>
         <width>81</width>
         <height>41</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label_9">
       <property name="geometry">
        <rect>
         <x>360</x>
         <y>90</y>
         <width>54</width>
         <height>21</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>宋体</family>
         <pointsize>11</pointsize>
        </font>
       </property>
       <property name="text">
        <string>温度</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_10">
       <property name="geometry">
        <rect>
         <x>360</x>
         <y>160</y>
         <width>54</width>
         <height>21</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>宋体</family>
         <pointsize>11</pointsize>
        </font>
       </property>
       <property name="text">
        <string>光强</string>
       </property>
      </widget>
      <widget class="QLCDNumber" name="ray">
       <property name="geometry">
        <rect>
         <x>400</x>
         <y>150</y>
         <width>81</width>
         <height>41</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label_11">
       <property name="geometry">
        <rect>
         <x>360</x>
         <y>230</y>
         <width>54</width>
         <height>21</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>宋体</family>
         <pointsize>11</pointsize>
        </font>
       </property>
       <property name="text">
        <string>人数</string>
       </property>
      </widget>
      <widget class="QLCDNumber" name="person">
       <property name="geometry">
        <rect>
         <x>400</x>
         <y>220</y>
         <width>81</width>
         <height>41</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label_12">
       <property name="geometry">
        <rect>
         <x>490</x>
         <y>91</y>
         <width>31</width>
         <height>21</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>℃</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_13">
       <property name="geometry">
        <rect>
         <x>490</x>
         <y>160</y>
         <width>31</width>
         <height>21</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>14</pointsize>
        </font>
       </property>
       <property name="text">
        <string>lx</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_14">
       <property name="geometry">
        <rect>
         <x>490</x>
         <y>230</y>
         <width>21</width>
         <height>21</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>个</string>
       </property>
      </widget>
      <widget class="QWidget" name="horizontalLayoutWidget">
       <property name="geometry">
        <rect>
         <x>360</x>
         <y>70</y>
         <width>160</width>
         <height>80</height>
        </rect>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_8">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_9"/>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="horizontalLayoutWidget_4">
       <property name="geometry">
        <rect>
         <x>360</x>
         <y>210</y>
         <width>161</width>
         <height>80</height>
        </rect>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_11"/>
      </widget>
      <widget class="QWidget" name="horizontalLayoutWidget_2">
       <property name="geometry">
        <rect>
         <x>360</x>
         <y>150</y>
         <width>160</width>
         <height>61</height>
        </rect>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_10"/>
      </widget>
      <widget class="QWidget" name="horizontalLayoutWidget_3">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>350</y>
         <width>301</width>
         <height>61</height>
        </rect>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_12">
        <item>
         <widget class="QPushButton" name="al">
          <property name="text">
           <string>A区</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="bl">
          <property name="text">
           <string>B区</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="cl">
          <property name="text">
           <string>C区</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
